README.md

```diff
@@ -1,4 +1,76 @@
 # TeamCity Build Opentelemetry Exporter Plugin
 
-A custom TeamCity plugin that contains a build listener to send OpenTelemetry trace data to an external endpoint
+A custom TeamCity plugin that registers a build listener to send build trace data to an opentelemetry collector external endpoint.
+
+## What is OpenTelemetry
+
+You can use OpenTelemetry to instrument, generate, collect, and export telemetry data (metrics, logs, and traces) for analysis in order to understand your software's performance and behavior. Create and collect telemetry data from your services and software, then forward them to a variety of analysis tools. For more information about OpenTelemetry go to https://opentelemetry.io/. Documentation for instrumentation for Java and other languages can be found [here](https://opentelemetry.io/docs/).
+
+## How to use this plugin
+
```

Comment:
Might need some words "install it to teamcity, stuff will appear magically in your honeycomb/zipkin/jaeger installation"

Reply:
added: 
"If you would like to visualize how you can better optimize your TeamCity builds and their dependency trees, this plugin exports TeamCity build pipeline data automatically to existing OpenTelemetry collector (such as honeycomb/zipkin/jaeger)."